OracleDatabase/SingleInstance/tests/runImageBuildTests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # LICENSE UPL 1.0
 #
-# Copyright (c) 1982-2017 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 1982-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should be `Copyright (c) 2017, 2020 Oracle and/or its affiliates.`

Reply:
Fixed